e2e/libs/node/aws.py

```diff
@@ -1,68 +0,0 @@
-import boto3
-import logging
-
-from botocore.exceptions import ClientError
-
-class EC2:
```

Comment:
`aws.py` has been removed.

And this abstraction will cause we have to implement every function for each platform. 
But do we really need to run power on/off test cases on every platform?

Even though we ideally expect to they can be run on every platform, it could be impossible to implement functions for each random platform users could use.

Details have been addressed here: https://github.com/longhorn/longhorn-tests/pull/1300#discussion_r1182275562

Reply:
oh, just noticed that aws.py and local_cluster.py will be removed in this PR. Yeah, having interface and abstraction only if it's needed.